styles/base/base.mcss

```diff
@@ -1,11 +1,14 @@
+html {
+  font-size: 12px
+}
+
 html, body {
   margin: 0
-  font: caption
```

Comment:
was this change intentional?

as far as i can tell, here's the difference:

before:

![Screenshot_20190615_165153](https://user-images.githubusercontent.com/719605/59547155-16134380-8f8e-11e9-881c-58dce0169130.png)

after:

![Screenshot_20190615_165118](https://user-images.githubusercontent.com/719605/59547156-20cdd880-8f8e-11e9-9e9f-0cd05ff2f082.png)